### DISCUSSION TITLE
Does this create standalone static (html+js, no server side) builds that can be deployed on IPFS?

### DISCUSSION THREAD

**Discussion by cryptotester** [scaffold-eth/scaffold-eth-2#873] [Q&A] [answered]
Does this create standalone static (html+js, no server side) builds that can be deployed on IPFS?

I need a system to generate a minimalist dapp that I could host on IPFS, that's why I was asking. Thank you.

**Comment by technophile-04** [comment, marked as answer]
Hey yes! It should most likely the steps to create an static export is: 

1. add `output: 'export'` in `packages/nextjs/next.config.js`
2. `cd packages/nexjts` and then run `yarn build` 
This will give you an static build of NextJs which you can most likely host on ipfs 

Adding some reference: 
1. https://nextjs.org/docs/app/building-your-application/deploying/static-exports#configuration
2. https://github.com/scaffold-eth/scaffold-eth-2/issues/785

**Reply by cryptotester** [reply]
Thank you, created a build, but seems to have issues (when running yarn start) such as this one, when e.g. I search for an address, or click an address (tested with the first address of PK 0 of hardhat):

 ⨯ Error: Page "/blockexplorer/address/[address]/page" is missing param "/blockexplorer/address/0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266" in "generateStaticParams()", which is required with "output: export" config.
    at DevServer.renderToResponseWithComponentsImpl (/scaffold-eth-2/packages/nextjs/node_modules/next/dist/server/base-server.js:1046:27) {
  page: '/blockexplorer/address/0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266'

**Reply by technophile-04** [reply]
> Thank you, created a build, but seems to have issues (when running yarn start) such as this one, when e.g. I search for an address, or click an address (tested with the first address of PK 0 of hardhat):
> 
> 
> 
>  ⨯ Error: Page "/blockexplorer/address/[address]/page" is missing param "/blockexplorer/address/0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266" in "generateStaticParams()", which is required with "output: export" config.
> 
>     at DevServer.renderToResponseWithComponentsImpl (/scaffold-eth-2/packages/nextjs/node_modules/next/dist/server/base-server.js:1046:27) {
> 
>   page: '/blockexplorer/address/0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266'

It seems you are trying to access the local blockexplorer, keep in mind that local blockexplorer should only be used in development, ideally before building static output you should build it such that `scaffold.config.ts` point to some testnet or mainnet.
